builtin_interfaces/Time stamp

float64 lat_deg
float64 lon_deg
float64 alt_m

float32 sog_mps
float32 sog_knots

float32 cog_deg
float32 heading_imu_deg
float32 heading_fused_deg
float32 yaw_rate_deg_s

bool   gnss_valid
int32  gnss_fix_quality
int32  gnss_num_sat
